### PR TITLE
Resize image height/width attrs, and fix typos

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of animals._

mouse in a shoe

## Description
_The commit messages say what you did; this should explain why and how._
- Fixed some typos
- Image height/width are now scaled down to match fallback-image size (1280×???)

### CODE REVIEW

**Comment by mirisuzanne** [comment]
@stacyk @jgerigmeyer the main feature here is in response to Stacy's comment about the home page image having height/width attributes set much larger than the image will ever actually display. 

- I think the simplest way to supply width/height manually is by using the actual image dimensions. Sometimes those numbers are much larger than we're actually referencing in image `src` or `srcset`.
- In most cases, browsers are only using these attributes in order to get the image ratio right while loading, so it's not important for the numbers to exactly match the output. But it's possible for the numbers to be used if CSS fails to load for any reason.
- There's no way for us to match exactly what's coming from the `srcset`, since that's dynamic, but we can more accurately reflect the fallback image `src` size. That's what this PR does - using either the fallback for a given type of image, or the default fallback of `1280`.

File: content/_includes/embed.macros.njk [hunk 1]
```diff
@@ -380,18 +380,21 @@ params:
...
 {%- set attrs = attrs | merge({
   'src': respSrc or src,
   'alt': alt,
   'srcset': imgSrcSet,
-  'width': width,
-  'height': height,
-  'sizes': (config.sizes[sizes] or sizes) if imgSrcSet else none
+  'width': resize.width,
+  'height': resize.height,
+  'sizes': configSizes or sizes
```
Comment (jgerigmeyer) [inline content/_includes/embed.macros.njk:397]: `sizes` is a string here -- is it (e.g. `img sizes="foobar"`) a meaningful fallback, or should we just use use `configSizes` or `none`?

File: content/_includes/embed.macros.njk [hunk 2]
```diff
@@ -380,18 +380,21 @@ params:
...
 {%- if not ('://' in src) -%}
+  {%- set fallback = config.sizes[sizes].fallback if config.sizes[sizes] else none -%}
   {%- set src = ('/assets/images/' + src) -%}
   {%- set imgSrcSet = srcset(src, width) -%}
   {%- set imgSrcSet = imgSrcSet if (imgSrcSet != "") else none -%}
-  {%- set respSrc = src | imgSuffix(config.fallback) -%}
+  {%- set respSrc = src | imgSuffix(fallback or config.fallback) -%}
+  {%- set configSizes = config.sizes[sizes].html if config.sizes[sizes] else none -%}
 {%- endif -%}
+{% set resize = width | imgSize(height, sizes) %}
```
Comment (jgerigmeyer) [inline content/_includes/embed.macros.njk:390]:
```suggestion
{%- set resize = width | imgSize(height, sizes) -%}
```

File: content/_includes/embed.macros.njk (hunk 1 above)
Comment (mirisuzanne) [reply content/_includes/embed.macros.njk:397]: While `sizes="foobar"` isn't useful, this is to allow either arbitrary or predefined sizes: 

- `configSizes` will grab e.g. `taxonomy.img.sizes['media'].html` (which is also a string) if `sizes == 'media'`
- Fallback to `sizes` means we can explicitly pass in any one-off sizes string like `'50vw'` or `'(min-width: 30em) 50vw, 100vw'`…

**Comment by mirisuzanne** [comment]
> I don't really understand what this solves or how it does it, but 👍 😂

@jgerigmeyer the quick version is two parts: 

- it updates the fallback `src` based on the image `sizes` (when predefined)
- it updates the actual `height`/`width` attributes to match the used fallback, while maintaining the proper image aspect ratio (when available).

File: content/index.md [hunk 3]
```diff
@@ -106,9 +106,10 @@ along with refactors for **integrated design systems,
 accessibility, performance,
 and long-term sustainability**.
 
-Since then OddBird has become an industry leaders --
+Since then OddBird has become an industry leader --
 from our work on Django, Sass, and Susy,
-to the **Mozilla Developer Channel**,
+to the Mozilla Developer Channel,
+**CSS Working Group**,
```
Comment (stacyk) [inline content/index.md:112]: 🔥

**Pull request merged**